dev/service-registry/devmode-scaladsl/src/main/scala/com/lightbend/lagom/devmode/internal/scaladsl/registry/LagomDevModeServiceRegistry.scala

```diff
@@ -79,8 +78,6 @@ case class ServiceRegistryService(uris: immutable.Seq[URI], acls: immutable.Seq[
 object ServiceRegistryService {
   def apply(uri: URI, acls: immutable.Seq[ServiceAcl]): ServiceRegistryService = ServiceRegistryService(Seq(uri), acls)
 
-  import UriFormat.uriFormat
```

Comment:
Hmm, these `uriFormat` imports could be necessary at runtime. Have you already tried runnnig the `scripted` tests? (you can do that locally using `sbt scripted`).

_The scripted tests haven't run in Travis yet for this PR because they are on a second stage_

Reply:
it will be fixed